backend/controllers/competitionController.js

```diff
@@ -44,15 +45,20 @@ const runCodeForNut = async (req, res) => {
     context, testCases, prependedCode, appendedCode, _id,
   } = task;
 
-  const vm = new VM();
+  const vm = new VM({
+    timeout: 10000,
```

Comment:
Er dette 10 sekunder?

Reply:
Jepp, tenkte 10 sekunder burde være nok for vårt scope hahah

Reply:
Ja enig

Reply:
Peder poengerte også at den er dynamisk, så vi kan faktisk sette den opp og ned avhengig av spesifikke testcases. Typ om vi har lyst til å kjøre én testcase med gigainput så kan vi det.

Reply:
Men i hovedsak var det for å stoppe `while (true) {}`

Reply:
Ja det var et godt poeng. Er jo fort gjort å slenge inn det feltet på en testcase og mate inn